.claude/skills/flow-define/SKILL.md

```
@@ -69,19 +69,24 @@ fi
-### STEP 1: Display Visual Indicators (MANDATORY - BLOCKING)
+### STEP 1: Display Visual Indicators
 
 **MANDATORY: You MUST use the Bash tool to run this provider check BEFORE displaying the banner. Do NOT skip it. Do NOT assume availability.**
 
 ```bash
 bash "${HOME}/.claude-octopus/plugin/scripts/helpers/check-providers.sh"
 ```
 
-**Use the ACTUAL results below. PROHIBITED: Showing only "🔵 Claude: Available ✓" without listing all providers.**
+Task status for the banner's `Tasks:` line, if the session has one:
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)